Description: Disable flaky checkerboard tests.
 .
 gitlab.gnome.org/GNOME/gtk/-/commit/a235b59089a7fe2d7957f20bcd2a67f9aef2f724
 gitlab.gnome.org/GNOME/gtk/-/commit/5e8e4130a51f8d1f599ee74360c5e9126ff40d48
 .
 "When textures are too large, we will scale them down to fit the max texture
 size, which will cause comparisons to fail."
Author: Nathan Pratta Teodosio <nathan.teodosio@canonical.com>
Forwarded: not-needed
Date: Tue, 22 Aug 2023 11:59:21 +0200

--- a/testsuite/gsk/meson.build
+++ b/testsuite/gsk/meson.build
@@ -11,7 +11,5 @@
 compare_render_tests = [
   'big-checkerboard',
-  'big-checkerboard-scaled-down',
   'big-checkerboard-scaled-down-nearest',
-  'big-checkerboard-scaled-down2',
   'blend-difference',
   'blend-modes',
